src/main/java/nl/tudelft/contextproject/script/Shot.java

```diff
@@ -24,8 +43,18 @@
      */
     public Shot(int num, Camera cam, Preset pres) {
         this.number = num;
+        this.shotId = null;
```

Comment:
It would be better to initialise shotId and description with empty strings instead of null.